docs/spec/typeddict.rst

```diff
@@ -284,24 +284,16 @@ refers to a dictionary object does not need to be supported, to simplify
 implementation.
 
 
-Type Consistency
-^^^^^^^^^^^^^^^^
+Assignability
+^^^^^^^^^^^^^
```

Comment:
```suggestion
Structural Assignability of TypedDicts
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
I don't see this change as an improvement. We are in the typed dict doc, so "of TypedDicts" seems redundant in a sub-heading (and could be applied to every sub-heading in the document.) And the heading "Assignability" accurately describes the topic of the sub-section. Adding more qualifiers to it ("Structural Assignability") is confusing to me because it implies that maybe there is another kind of assignability for typed dicts which is discussed elsewhere.

Reply:
I don't think it's another kind of assignability? This section should apply the general concept of assignability to TypedDicts.